packages/block-editor/src/components/block-list/style.scss

```diff
@@ -628,6 +628,15 @@
 	}
 }
 
+// Hide the popover block editor list while dragging.
+// Using a hacky animation to delay hiding the element so that dragging can still work.
```

Comment:
Maybe mention that if the element is hidden immediately, dragging won't work, and this is why a delay is required.